main.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include <iomanip>
+
+int main() {
+
+    std::cout << "JOE'S ACME SOFTWARE, INC. STOCK VALUES\n\n"; // just some header text
+
+    //stock purchase calculations
+    const int SHARES_BOUGHT = 1000;
```

Comment:
So one thing we never covered is that by current general style and our target google style named constants should be declared at the top of our scope.
https://google.github.io/styleguide/cppguide.html#Declaration_Order